hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
```

Comment:
because it breaks the contract right so whats the folder structure then look like in bundles ya know